sys/net/ieee802154.txt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2021 Christian Amsüss <chrysn@fsfe.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_ieee802154 IEEE 802.15.4
+ * @ingroup     net
+ * @brief       IEEE 802.15.4 components
+ *
+ * General introduction
+ * --------------------
+ *
+ * The IEEE 802.15.4 standard describes radio communication for low-rate local
+ * wireless networks, operating in different frequency bands, modulations and
+ * MACs; all of these have been extended over time in revisions of the
+ * standard. It also describes topologies and roles of devices (eg. a PAN
+ * coordinator that is a FFD (Full Function Device) and RFDs (Reduced Function
+ * Devices) and clusters that use different PAN IDs to keep their traffic
+ * apart), without prescribing how they are formed or managed.
+ *
+ * Common frequencies used are the 868 MHz band (one channel, in Europe), the
+ * 915 MHz band (10 channels, in America) and the 2.4GHz band (16 channels);
+ * several more are defined between 169MHz and 9GHz.
+ *
+ * The standard describes several MACs that govern when to send and to listen;
+ * the choice of the MAC involves the trade-offs between power consumption,
+ * complexity and latency, as it governs how much time devices can spend with
+ * their radios powered down. Some MACs support channel hopping to avoid
+ * frequencies used by others. Common MACs are:
+ *
+ * * nonbeacon-eanbled: devices ask the coordinator to send a beacon in short
+ *   transmissions, then the coordinator sends a beacon guiding further
+ *   transmissions
```

Comment:
In NBE the beacon is only used for network discovery. Maybe the reference point to Indirect Transmission, in which the RFD sends a Data Request frame to the coordinator in order to poll pending data. However this is not mandatory, as nodes can also decide to be permanently on (e.g non-battery powered nodes).